crates/aptos/src/main.rs

```diff
@@ -5,12 +5,14 @@
 
 #![forbid(unsafe_code)]
 
-use aptos::Tool;
+use aptos::{move_tool, Tool};
 use clap::Parser;
 use std::process::exit;
 
 #[tokio::main]
 async fn main() {
+    // Register hooks
+    move_tool::register_package_hooks();
```

Comment:
I wonder, why not just in the MoveTool?

Reply:
This should only happen once per process. We could indirectly intialize it via a `Lazy<()>` in the move tool, but it seems cleaner to do this there. There are other hooks on the move system we may need over time.